rfqmsg/accept.go

```diff
@@ -0,0 +1,225 @@
+package rfqmsg
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"io"
+
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/routing/route"
+	"github.com/lightningnetwork/lnd/tlv"
+)
+
+const (
+	// Accept message type field TLV types.
+
+	TypeAcceptID          tlv.Type = 0
+	TypeAcceptAskingPrice tlv.Type = 2
+	TypeAcceptExpiry      tlv.Type = 4
+	TypeAcceptSignature   tlv.Type = 6
```

Comment:
As mentioned, the sig may not be 100% necessary, but we can keep it on for now until we get further into pipeline.